failed to build 0.37.0

👋 running into some build failure when building 0.37.0 release 

```
  error[E0432]: unresolved import `chrono`
   --> src/cli/src/cmd/ls.rs:9:5
    |
  9 | use chrono::{Local, TimeZone};
    |     ^^^^^^ use of unresolved module or unlinked crate `chrono`
    |
    = help: if you wanted to use a crate named `chrono`, use `cargo add chrono` to add it to your `Cargo.toml`
```

build log, https://github.com/Homebrew/homebrew-core/actions/runs/17470728961/job/49624431646?pr=236163#step:3:1230
relates to https://github.com/Homebrew/homebrew-core/pull/236163

--- a/oxen-rust/src/cli/Cargo.toml
+++ b/oxen-rust/src/cli/Cargo.toml
@@ -6,6 +6,7 @@ edition = "2021"
 [dependencies]
 async-trait = "0.1.80"
 bytesize = "1.3.0"
+chrono = "0.4.30"
 clap = { version = "4.2.7", features = ["cargo", "derive"] }
 colored = "2.0.0"
 dialoguer = "0.11.0"

